modules/ibi-create-config-iso.adoc

```diff
@@ -0,0 +1,316 @@
+
+.Procedure
+
+. Create the `ibi-ns` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace ibi-ns
+----
+
+. Create the `Secret` resource for your image registry:
+
+.. Create a YAML file that defines the `Secret` resource for your image registry:
++
+.Example `secret-image-registry.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ibi-image-pull-secret
+  namespace: ibi-ns
+stringData:
+  .dockerconfigjson: <base64-docker-auth-code> <1>
+type: kubernetes.io/dockerconfigjson
+----
+<1> You must provide base64-encoded credential details. See the "Additional resources" section for more information about using image pull secrets.
+
+.. Create the `Secret` resource for your image registry by running the following command:
++
+[source,terminal]
+----
+$ oc create -f secret-image-registry.yaml
+----
+
+. Optional. Configure static networking for the host:
+
+.. Create a `Secret` resource containing the static network configuration in `nmstate` format:
++
+.Example `host-network-config-secret.yaml` file
+[source,yaml]
+----
+apiVersion: v1 
+kind: Secret
+metadata:
+ name: host-network-config-secret <1>
+ namespace: openshift-machine-api
+type: Opaque
+stringData:
+ nmstate: | <2>
+  interfaces:
+    - name: ens1f0 <3>
+      type: ethernet
+      state: up
+      ipv4:
+        enabled: true
+        address:
+          - ip: 10.6.85.8
+            prefix-length: 24
+        dhcp: false <4>
+        autoconf: false <5>
+      ipv6:
+        enabled: false
+  dns-resolver:
+    config:
+      server:
+        - 10.6.73.2 <6>
+        - 10.6.73.4
+  routes:
+    config: <7>
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 10.6.85.254
+        next-hop-interface: ens1f0
+        table-id: 254
+----
+<1> Specify the name for the `Secret` resource.
+<2> Define the static network configuration in `nmstate` format.
+<3> Specify the name of the interface on the host. The name of the interface must match the actual NIC name as shown in the operating system. To use your MAC address for NIC matching, set the `identifier` field to `mac-address`.
+<4> You must specify `dhcp: false` to ensure `nmstate` assigns the static IP address to the interface.
+<5> You must specify `autoconf: false` to ensure `nmstate` assigns the static IP address to the interface.
+<6> Specify one or more DNS servers that the system will use to resolve domain names.
+<7> In this example, the default route is configured through the `ens1f0` interface to the next hop IP address `10.6.85.254`.
+
+. Create the `BareMetalHost` and `Secret` resources:
+
+.. Create a YAML file that defines the `BareMetalHost` and `Secret` resources:
++
+.Example `ibi-bmh.yaml` file
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: ibi-bmh <1>
+  namespace: openshift-machine-api
```

Comment:
It's more common to see this BMH created in the same namespace as the other resources (ibi-ns in this example)